content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -49,9 +49,16 @@ By default, JVM metrics from your application are sent to the Datadog Agent over
 {{% /tab %}}
 {{% tab "Python" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+Python metrics collection can be enabled with one environment parameter when running with `ddtrace-run`:
 
-[1]: /help
+* Environment Variable: `DD_RUNTIME_METRICS_ENABLED=true`
+
+Python metrics can be viewed in correlation with your Python services. See the [Service page][1] in Datadog.
+
+**Note**: For the runtime UI, `ddtrace` >= [`0.24.0`][2] is supported.
```

Comment:
ddtrace-py we mean?

Reply:
The repo is `dd-trace-py`, but you install it as `ddtrace`